org.civicrm.bootstrapcivicrm/scss/components/_page.scss

```diff
@@ -3,23 +3,45 @@
     @media (min-width: $screen-md-min) {
         overflow: hidden;
 
-        // Override of the grid widths
-        > [class*="col-"] {
+        > .row {
 
-            &:first-child {
-                width: 12.775%;
-            }
+            > [class*="col-"] {
 
-            &:last-child {
-                width: 87.2%;
+                &:first-child {
+                    max-width: $crm-page-sidebar-max-width + $grid-gutter-width;
+                }
             }
         }
 
+        // On medium/large viewports the main block must
+        // be centered and should not stretch after a certain point
+        .crm_page__main {
+            margin: 0 auto;
+            max-width: $crm-page-main-max-width;
```

Comment:
Yup, but if it could be in another media query, like: 

    @media (min-width: $crm-page-main-max-width) {
         width: $crm-page-main-max-width;
         margin: 0 auto;
     }

I think this way it would better show what you're intending to do (and therefore the comment wouldn't be needed).